bottlecap/src/config/mod.rs

```diff
@@ -400,9 +396,11 @@ fn fallback(config: &Config) -> Result<(), ConfigError> {
         ));
     }
 
-    if config.serverless_appsec_enabled || config.appsec_enabled {
-        log_fallback_reason("appsec_enabled");
-        return Err(ConfigError::UnsupportedField("appsec_enabled".to_string()));
+    if config.serverless_appsec_enabled {
+        log_fallback_reason("serverless_appsec_enabled");
```

Comment:
```suggestion
        log_fallback_reason("appsec_enabled");
```

you can leave this one as `appsec_enabled` since this is reported as a metric